### ISSUE TITLE
ambiguous training validation loss 

### ISSUE THREAD

**Issue by TieLeeyoung** [rizkiarm/LipNet#47]
hello, i'm running the model, but i'm getting a bad loss, in the sixth epoch i got these scores:
[Epoch 6] Out of 256 samples: [CER: 25.898 - 0.910] [WER: 7.492 - 1.457] [BLEU: 0.314 - 0.314]
Are these scors meaningful? or the model overfit, and what's is the bleu score? is it the accuracy in speech recognition

**Comment by qslvhw** [comment]
I have the same question. I only got loss and validation loss after every epoch. How to see the accuracy?

**Comment by rizkiarm** [comment]
The performance of the model is indicated by Word Error Rate (WER), Character Error Rate (CER), and BLEU score. You can find out more about that by reading the paper. Based on the scores that you have got, it seems that your model was not learning. Please check whether you made some mistake in your setup and make sure that you provide enough (and correct) examples for the model.

I will close this issue for now. Please re-open it if you think that you can provide more detail about your problem and have tried your best to fix it, as I think it is not the issue with the program.
